src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -343,6 +343,8 @@ class ArnoldOutput : public IECore::RefCounted
 			// supported by Arnold.
 
 			m_data = output->getData();
+			m_lpeName = name.string() + "__LPE__";
```

Comment:
I need to make sure that LPE names are completely unique.  If you create a display named "diffuse" using an LPE, it would currently reference an LPE named "diffuse__LPE__".  If it tried to reference an LPE named "diffuse", there's a chance Arnold could have a default shader output named "diffuse", and we would get that instead of our LPE.  I'm happy for alternate naming conventions if you have an idea.  I guess usually in Gaffer we would just use a giant hash value to name something that's been generated, but diffuse__LPE__ is more legible in the ass than a MurmurHash.